### DISCUSSION TITLE
Help with decline informed consent in cognirion.run and prolific

### DISCUSSION THREAD

**Discussion by orduek** [jspsych/jsPsych#2929] [Q&A] [answered]
Hi,
I want to use cognition.run informed consent option, but not sure what I should put in the decline URL.
I intend to use Prolific to find participants, and I know how to embed the "back to prolific" URL at the end of the experiment. But should I use the same URL for declined informed consent?

**Comment by ChristopheBossens** [comment, marked as answer]
Hi,

I just found out that there is an option to generate custom completion codes (the default one is related to completing and approving the study). So you have to create a completion code for the case where the participant does not accept the consent form, and use that URL to redirect. The effect of this redirect is that the participant will be asked to return their submission.

https://researcher-help.prolific.co/hc/en-gb/articles/5805656558236-Custom-Completion-Codes

**Comment by orduek** [comment]
So simple :)
Thanks for the help.
